policies/coding-style.md

```diff
@@ -548,17 +549,17 @@ Functions can return values of many different kinds, and one of the
 most common is a value indicating whether the function succeeded or
 failed. Usually this is:
 
-    1: success
-    0: failure
+* 1: success
+* 0: failure
 
 Sometimes an additional value is used:
 
-    -1: something bad (e.g., internal error or memory allocation failure)
+* -1: something bad (e.g., internal error or memory allocation failure)
 
 Other APIs use the following pattern:
 
-    >= 1: success, with value returning additional information
-    <= 0: failure with return value indicating why things failed
+* \>= 1: success, with value returning additional information
+* <= 0: failure with return value indicating why things failed
```

Comment:
These appear in a block quote because of the starting `>`. Adding bullets and escaping the `>` formats it as above.